privacyidea/static/components/token/views/token.enroll.html

```diff
@@ -174,11 +174,29 @@ <h4 translate>Assign token to user</h4>
     '/static/components/token/views/token.enrolled.' + form.type + '.html' + fileVersionSuffix">
     </ng-include>
 
+    <!-- Verify Token enrollment -->
+    <div ng-show="enrolledToken.rollout_state === 'verify'">
+        <p translate>The token was enrolled, but you still need to verify it before you can use it!</p>
+        <div class="form-group">
+            <label for="verifyResponse" translate>{{ enrolledToken.verify.message }}</label>
+            <input type="text" ng-model="verifyResponse"
+                   autofocus
+                   class="form-control"
+                   name="verifyResponse">
+        </div>
+        <div class="text-center">
+            <button type="button" ng-click="sendVerifyResponse()"
+                    class="btn btn-primary" translate>Verify Token</button>
+        </div>
+    </div>
```

Comment:
After verifying the token, we are still on the token.enrolled page (but without the seeds and a working googleurl link).
Maybe after the verify we switch back to the previous/list view.

Reply:
won't fix.

Reply:
Yes, but the `otpauth://` link is invalid. We should at least hide the "Click here..." part after the verification

Reply:
Will do, looks good.